src/pages/view-test-result/components/faults-data-row/__tests__/faults-data-row.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { FaultsDataRowComponent } from '../faults-data-row';
+import { FaultSummary } from '../../../../../shared/models/fault-marking.model';
+
+describe('FaultsDataRowComponent', () => {
```

Comment:
can we have a test or two around the default value for minimum fault count. I can see that being changed and breaking stuff in the future, without a developer realizing

Reply:
Sure, what do you think about this?